surefire-its/src/test/java/org/apache/maven/surefire/its/jiras/Surefire1787JUnit5IT.java

```diff
@@ -93,6 +93,18 @@ public void selectJUnit5()
                 "Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider" );
     }
 
+    @Test
+    public void selectJUnit5UsingConfiguredProvider()
```

Comment:
Fair enough. I've opened a separate https://issues.apache.org/jira/browse/SUREFIRE-2036 since the original regression still exists, and reworked this PR against that issue: e898f07ebe7e5ce29f6bf2c6070ed28370c95805